### DISCUSSION TITLE
Open question: Is "No-2s Tenney height" too non-standard?

### DISCUSSION THREAD

**Discussion by m-yac** [m-yac/xen-calc#15] [General]
Currently the default sort for best rational approximations is no-2s Tenney height (remove all factors of 2 then compute the Tenney height), or equivalently, no-2s Benedetti height (remove all factors of 2 then compute the Benedetti height).

I find this sort gives the most intuitive results to me, but I couldn't find anyone else who had written about this height function. Is this too non-standard of a height function to have as the default? Is there some catch when using a height function like this that I'm not aware of? I'd love to hear anyone's thoughts.

**Comment by Xenwolf** [comment]
The algorithm you describe seems to correspond to the definition of [*Kees height*](https://en.xen.wiki/w/Kees_height "Kees height - Xenharmonic Wiki").

BTW: What do you think of the “feature request” on [Talk:Monzo #Coping with large amounts of zeros](https://en.xen.wiki/w/Talk:Monzo#Coping_with_large_amounts_of_zeros "Talk:Monzo - Xenharmonic Wiki Coping with large amounts of zeros") to use exponents for heavily repetitive zeros in cases like [67/61](https://www.yacavone.net/xen-calc/?q=67/61).
